app/src/shared/components/ImageUpload/index.jsx

```diff
@@ -53,9 +53,10 @@ const ImageUpload = ({
             setUploaded(false)
 
             if (setImageToUpload) {
-                setImageToUpload(((Object.assign({}, image, {
+                // $FlowFixMe property `preview` is missing in  `File`.
+                setImageToUpload(Object.assign(image, {
                     preview: imagePreview,
-                }): any): DropzoneFile))
+                }))
```

Comment:
@juhah it may not be related to type stuff but to the way I changed `Object.assign`. Can you try this?
```js
setImageToUpload(((Object.assign(image, {
    preview: imagePreview,
}): any): DropzoneFile))
```

The difference: instead of `Object.assign({}, object, {…` I used `Object.assign(image, {…`.

lmk if it works

Reply:
I changed it thinking "well, let's leave the original `image` unchanged and return a fresh object istead". Apparently it doens't work like this.  XD

Reply:
Yep, it was the first empty parameter, not types that caused it.. now it works again 👍